Geena - geena-panzitta/GLP_week4assignment.do

```diff
@@ -0,0 +1,289 @@
+********************************************************************************
+* Econ 490: Week 4
+* Week 4 Assignment
+* Geena Panzitta
+* October 2, 2022
+********************************************************************************
+
+set more off
+clear
+
+//change working directory here
+cd "/Users/geenapanzitta/Library/CloudStorage/Box-Box/Econ490_Fall2022/Week4/04_assignment/data"
+
+/*******************************************************************************
+Q1.
+*******************************************************************************/
+{
+	
+use village_pixel.dta, clear
+
+}
+/*******************************************************************************
+Q1. a
+*******************************************************************************/
+{
+
+gen pixel_consistent = .
+label variable pixel_consistent "Are payouts consistent without pixel?"
```

Comment:
Nice use of variable labels-- you reminded me that I should probably add this to my code as well!